cime_config/cesm/machines/Makefile

```diff
@@ -776,8 +770,8 @@ cleanrof:
 	cd $(EXEROOT)/rof/obj ; $(RM) -f *.o *.mod
 
 cleanlnd:
-	$(RM) -f $(LIBROOT)/liblnd.a
-	cd $(EXEROOT)/lnd/obj ; $(RM) -f *.o *.mod
+	$(RM) -f $(LNDLIBDIR)/$(LNDLIB)
```

Comment:
Okay, yellowstone-intel aux_clm45 test suite looks good (run from clm4_5_8_r171 with cime updated to this branch)